sample-authn/src/main/java/net/trajano/ms/example/authn/AuthnResource.java

```diff
@@ -1,19 +1,24 @@
 package net.trajano.ms.example.authn;
 
+import java.net.URI;
 import java.util.List;
 
 import javax.annotation.security.PermitAll;
 import javax.ws.rs.Consumes;
+import javax.ws.rs.FormParam;
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
-import javax.ws.rs.core.Context;
-import javax.ws.rs.core.HttpHeaders;
-import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.MultivaluedHashMap;
-import javax.ws.rs.core.MultivaluedMap;
-import javax.ws.rs.core.Response;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.*;
 
+import net.trajano.ms.auth.token.ErrorCodes;
+import net.trajano.ms.common.oauth.OAuthTokenResponse;
+import net.trajano.ms.core.CryptoOps;
+import org.jose4j.jws.JsonWebSignature;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.jose4j.jws.JsonWebSignature'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)